### PR TITLE
ini lookup - catch and handle duplicate key and missing section errors

### PR DESCRIPTION
**Pull request by samdoran**
##### SUMMARY

Partial fix for #74601.

Catch and handle duplicate key and missing section errors. Currently they raise exceptions which we do not handle.

##### ISSUE TYPE

- Bugfix Pull Request

##### COMPONENT NAME

`lib/ansible/plugins/lookup/ini.py`

### CODE REVIEW

File: lib/ansible/plugins/lookup/ini.py [hunk 1]
```diff
@@ -15,7 +15,10 @@
       - "You can also read a property file which - in this case - does not contain section."
     options:
       _terms:
-        description: The key(s) to look up
+        description:
+            The key(s) to look up. On Python 2, key names are case B(insensitive).
+            In Python 3, key names are case B(sensitive). Duplicate key names found
```
Comment (bcoca) [inline lib/ansible/plugins/lookup/ini.py]: this wont run on py2 anymore .. specially with the string syntax below

you might want to make separate PR to port the py2 info back

**Review by bcoca — APPROVED** [review]
description aside, rest lgtm

File: lib/ansible/plugins/lookup/ini.py (hunk 1 above)
Comment (samdoran) [reply lib/ansible/plugins/lookup/ini.py]: Sounds good. I will add that note in the backport.
